libgadget/exchange.c

```diff
@@ -47,10 +47,10 @@ typedef struct {
  * exchange particles according to layoutfunc.
  * layoutfunc gives the target task of particle p.
 */
-static int domain_exchange_once(ExchangePlan * plan, int do_gc, MPI_Comm Comm);
-static void domain_build_plan(ExchangeLayoutFunc layoutfunc, const void * layout_userdata, ExchangePlan * plan);
-static int domain_find_iter_space(ExchangePlan * plan);
-static void domain_build_exchange_list(ExchangeLayoutFunc layoutfunc, const void * layout_userdata, ExchangePlan * plan, MPI_Comm Comm);
+static int domain_exchange_once(ExchangePlan * plan, int do_gc, struct part_manager_type * PartManager, struct slots_manager_type * SlotsManager, MPI_Comm Comm);
```

Comment:
Perhaps use part_manager as a variable name? Then you can add the typedef. It seems to be more conventional using CamelCase for type names.

I found sed helps in massive replacements like this:
```
sed --in-place 's;\<PartManager\>;part_manager;g' *.[ch]
sed --in-place 's;struct part_manager_type;PartManager;g' *.[ch]
```

Reply:
part_mgr if part_manager looks too long?

Reply:
or even `pman` and `sman`.